src/shared/parameter/config_definitions/defense_play_config.yaml

```diff
@@ -0,0 +1,6 @@
+- double:
+    name: immediate_threat_distance
+    min: 0.0
+    max: 9.0
+    value: 4.5
```

Comment:
I'm not convinced that 4.5 is good, because if an enemy is at the halfway line, there are lots of good passes that we should be blocking, rather than swarming the robot

Reply:
I think 6 should be enough, it covers the entire friendly area and a decent bit of the enemy area. By swarming the enemy robot we are also stopping that robot from passing